cvmfs/mountpoint.h

```diff
@@ -219,7 +220,7 @@ class FileSystem : SingleCopy, public BootFactory {
   bool LockWorkspace();
   bool SetupCrashGuard();
   bool CreateCache();
-  bool SetupQuotaMgmt();
+  bool SetupQuotaMgmt(cache::CacheManagerIds mgr_type);
```

Comment:
It seems more natural as a member of `Filesystem`. The cache and quota setup is there